plugins/woocommerce/tests/e2e-pw/tests/merchant/product-reviews.spec.js

```diff
@@ -60,4 +62,66 @@ test.describe( 'Products > Reviews', () => {
 			page.getByRole( 'link', { name: reviewerEmail } )
 		).toBeVisible();
 	} );
+
+	test( 'can edit product review', async ( { page } ) => {
+		await page.goto(
+			`wp-admin/edit.php?post_type=product&page=product-reviews`
+		);
+		await expect(
+			page.getByRole( 'cell', { name: productReview } )
+		).toBeVisible();
+
+		//Hover over the product review to have 'Edit' displayed
+		await page.hover( '.comment-text' );
+
+		// Select Quick Edit, edit the review and save
+		await page.getByRole( 'button', { name: 'Quick Edit' } ).click();
+		await page.locator( '.wp-editor-area' ).first().fill( updatedReview ); //updatedReview
+		await page.getByRole( 'button', { name: 'Update Comment' } ).click();
+		await page.waitForTimeout( 2000 );
+
+		// Verify that the edited comment is there
+		const commentTextElement = page.locator( '.comment-text' );
+		await commentTextElement.waitFor( { state: 'visible' } );
+		const updatedComment = await commentTextElement.innerText(); //updatedComment
+		await expect( updatedComment ).toBe( updatedReview ); //updatedComment updatedReview
+	} );
+
+	test( 'can delete product review', async ( { page } ) => {
+		await page.goto(
+			`wp-admin/edit.php?post_type=product&page=product-reviews`
+		);
+		await expect(
+			page.getByRole( 'cell', { name: updatedReview } )
+		).toBeVisible();
+
+		//Hover over the product review to have 'Edit' displayed
+		await page.hover( '.comment-text' );
+
+		// Select Trash action, check confirmation prompt and undo
+		await page.getByRole( 'button', { name: 'Trash' } ).click();
+		await page.waitForTimeout( 2000 );
```

Comment:
It's not a good practice to use waitForTimeout. Instead, you should wait for something to happen in the app, like for an element to be displayed. 
In fact that's exactly what the next line does, it waits for element with the specific message, so this wait is not helping at all, it only adds 2 seconds to the test execution time.